sklearn/linear_model/tests/test_bayes.py

```diff
@@ -110,6 +111,22 @@ def test_std_bayesian_ridge_ard_with_constant_input():
         assert_array_less(y_std, expected_upper_boundary)
 
 
+def test_regression_issue_10128():
+    # this ARDRegression test throws a `ValueError` on master, commit 5963fd2
+    np.random.seed(752)
+    n = 100
+    d = 10
+    n_samples = 96
+    n_features = 9
+    X = sparse_random_matrix(n, d, density=0.10).toarray()
+    X, y = X[:, :n_features], X[:, -1]
+    X_train, y_train = X[:n_samples], y[:n_samples]
+    X_test = np.zeros((1, n_features))
+    clf = ARDRegression()
+    clf.fit(X_train, y_train)
+    clf.predict(X_test, return_std=True)
+
```

Comment:
Maybe checking the value of the coef would be meaningful in this case

Reply:
done

Reply:
I check for the correct shape of `sigma_` now